dash-pipeline/bmv2/dash_pipeline.p4

```diff
@@ -182,7 +183,9 @@ control dash_ingress(inout headers_t hdr,
     @name("inbound_routing|dash_vnet")
     table inbound_routing {
         key = {
+            meta.eni_id: exact @name("meta.eni_id:eni_id");
             hdr.vxlan.vni : exact @name("hdr.vxlan.vni:VNI");
+            hdr.ipv4.src_addr : ternary @name("hdr.ipv4.src_addr:sip");
```

Comment:
So there is a src_addr in both the inbound_routing and pa_validation. So even if the inbound routing rule is for a specific src_addr we still expect a pa_validation entry for the same ?

Reply:
@vmytnykx , inbound routing is not LPM - Please refer here - https://github.com/Azure/DASH/blob/main/documentation/general/design/dash-sonic-hld.md#22-inbound-packet-processing-pipeline. @KrisNey-MSFT , we may want to correct the vnet-to-vnet-service.md section.